cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/1_create_datasource.spec.js

```diff
@@ -23,6 +23,32 @@ const miscUtils = new MiscUtils(cy);
 const username = Cypress.env('username');
 const password = Cypress.env('password');
 
+export const CreateDataSourceBasicAuth = () => {
+  miscUtils.visitPage(
+    'app/management/opensearch-dashboards/dataSources/create'
+  );
+
+  cy.get('[data-test-subj="createDataSourceButton"]').should('be.disabled');
+  cy.get('[name="dataSourceTitle"]').type('9201');
+  cy.get('[name="endpoint"]').type('https://localhost:9202');
```

Comment:
is it a typo? I see the endpoint defined in createDataSourceNoAuth is 9201?

Reply:
It wasn't a typo - it was for creating another instance with security on 9202. I removed it and added it as a command in: [46d3442](https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1146/commits/46d3442825d45ef318d72cd3a45aac3f4c0078da)